src/components/BuildBaronAndAnnotations/AnnotationTicketsTable.tsx

```diff
@@ -165,16 +167,38 @@ export const AnnotationTicketsTable: React.FC<Props> = ({
     moveAnnotation({ variables: { annotationId, apiIssue, isIssue } });
   };
 
+  // SCROLL TO added Issue
+  const rowRef = useRef<HTMLSpanElement>(null);
+
+  useEffect(() => {
+    if (selectedRowKeys && rowRef.current) {
+      rowRef.current.scrollIntoView({
```

Comment:
we should adjust when this function is called so we do not scroll when the mutation fails 

https://user-images.githubusercontent.com/10734386/104774683-1ca70580-5745-11eb-9c32-89c2d3d47534.mov

Reply:
Thank you for finding this!

Reply:
I added window.scrollTo(0, 0); to onError.

Reply:
I dont think we should do that. Since all thats doing is adding an additional scroll to mask the original scroll. It shouldn't scroll if there isn't anything to scroll to

Reply:
I think adding apollo error variable to the condition in this useEffect could tell us when it's appropriate to scroll

Reply:
I tried nulling selectedRowKey when the mutation fails, it doesn't work. I'll try using the apollo error.